dev/bots/hello_world_ftl.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+pushd ../../examples/hello_world
+
+# To generate gradle wrapper.
+# TODO: Find a smarter way to get gradlew without building the app.
+../../bin/flutter build appbundle
+
+pushd android
+
+# Build the app and instrumentation apks.
+./gradlew clean
+./gradlew assembleAndroidTest
+./gradlew assembleDebug -Ptarget="$(pwd)/../test/hello_adapter_test.dart"
+
+# Run on firebase test lab.
+gcloud firebase test android run \
+  --type=instrumentation \
+  --app="../build/app/outputs/apk/debug/app-debug.apk" \
+  --test="../build/app/outputs/apk/androidTest/debug/app-debug-androidTest.apk" \
+  --device model=Pixel2,version=28,locale=en,orientation=portrait \
+  --project tong-hello1
```

Comment:
this should use the flutter-infra project  as well as the storage bucket from it.

Also, as discussed offline, it would probably be good to create a dart program to manage these, since now there are at least two in this repo.